src/lightly_train/_transforms/transform.py

```diff
@@ -119,7 +119,7 @@ def cast_list_to_tuple(cls, value: int | Sequence[int]) -> int | tuple[int, int]
             and all(isinstance(v, int) for v in value)
         ):
             value = tuple(value)
-            assert isinstance(value, tuple)
+            assert len(value) == 2
```

Comment:
Old assert doesn't make sense because we just created the tuple. Check for len 2 is needed however.